### PR TITLE
[docker] fix docker-run-dynamic

### PR DESCRIPTION
**Pull request by davidiw**
the epxression was flawed in that it didn't consider other deployments
and broke them.

## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: docker/validator-dynamic/docker-run-dynamic.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ fi
     ${params[@]}
 
 
-if [ -n "${CFG_FULLNODE_SEED}" -a "${CFG_ENABLE_MGMT_TOOL}" = false ]; then # We have a full node seed, add fullnode network
+if [[ -n "${CFG_FULLNODE_SEED}" && ("${CFG_ENABLE_MGMT_TOOL}" = false || -z "${CFG_ENABLE_MGMT_TOOL}") ]]; then
```
Comment (sausagee) [inline docker/validator-dynamic/docker-run-dynamic.sh:45]: nit - add a comment that a fullnode network is created iff its seed is defined and management tool is not enabled.

Comment (davidiw) [reply docker/validator-dynamic/docker-run-dynamic.sh:45]: Sounds good!

**Comment by davidiw** [comment]
well this is great, I fixed it locally but didn't actually push your "nit"... I'm not sure it is worth it now, since I am going to delete this file really soon.

**Pull request closed without merging**